packages/teams/src/remove-users-from-team.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  IRemoveGroupUsersResult,
+  removeGroupUsers,
+} from "@esri/arcgis-rest-portal";
+import { IAuthenticationManager } from "@esri/arcgis-rest-request";
+
+/**
+ * Remove N users from team. Acts as a wrapper around removeGroupUsers from arcgis-rest-portal
```

Comment:
Generally I don't think we need wrappers if the direct call does the exact same thing.

Reply:
@ajturner  I'd thought we wanted Hub.js to be "complete" and to use Hub Terminology - thus `removeUsersFromTeam` vs expecting a user to know to import some other fn from REST-JS?